pkg/sgpp/constants.go

```diff
@@ -0,0 +1,15 @@
+package sgpp
+
+type EnforcingMode string
+
+const (
+	EnforcingModeStrict   EnforcingMode = "strict"
+	EnforcingModeStandard EnforcingMode = "standard"
+)
+
+const (
+	// DefaultEnforcingMode is the default enforcing mode if not specified explicitly.
+	DefaultEnforcingMode EnforcingMode = EnforcingModeStrict
```

Comment:
wonder if we should rename "strict" -> "default" instead and "standard" -> something like "partial".  Since standard tends to make you feel it's the standard when actually it's not. @M00nF1sh your call, just my thought.

Reply:
this was proposed by our PM. we prefer to use "standard" as the default mode in the future, that's why we have the naming. 
I'm with him as "standard" can be interpreted as similar behavior with "standard pods"(i.e. pods without security group)